builtins/amp-pixel.js

```diff
@@ -95,6 +107,49 @@ export class AmpPixel extends BaseElement {
   }
 }
 
+/**
+ * @param {!Element} parentElement
+ * @param {string} src
+ * @returns {!Element}
+ */
+function createNoReferrerPixel(parentElement, src) {
+  if (isReferrerPolicySupported()) {
+    return createImagePixel(src, true);
+  } else {
+    // if "referrerPolicy" is not supported, use iframe wrapper
+    // to scrub the referrer.
+    const iframe = createElementWithAttributes(
+        /** @type {!Document} */ (parentElement.ownerDocument), 'iframe', {
+          src: `javascript: '<img src="${src}">'`,
```

Comment:
Question: Will this work in prod with our CSP preventing javascript URLs?

I don't think do. Can you create an `about:blank` frame instead and append the image pixel instead?